### ISSUE TITLE
Unsupported shape calculation for operator customModel

### ISSUE THREAD

**Issue by mbroedel** [onnx/onnxmltools#379]
Hi, I'm trying to convert a sound classification model created with TuriCreate from CoreML to ONXX with the following code:
```python
import coremltools
import onnxmltools

input_coreml_model = 'model.mlmodel'
output_onnx_model = 'model.onnx'
coreml_model = coremltools.utils.load_spec(input_coreml_model)
onnx_model = onnxmltools.convert_coreml(coreml_model)
onnxmltools.utils.save_model(onnx_model, output_onnx_model)
```
I get `ValueError: Unsupported shape calculation for operator customModel` on line 7.

**Comment by jiafatom** [comment]
Can you share the model, please?

**Comment by mbroedel** [comment]
[Here you go](https://drive.google.com/file/d/1Xl9hn19UJBGwwkAjWqEHzrR4GIAbnwOy/view?usp=sharing)

**Comment by jiafatom** [comment]
You need specify the function `customModel` as a parameter `custom_conversion_functions` in `convert_coreml`, otherwise it does not know what `customModel` is.

**Comment by mbroedel** [comment]
Oh okay, that makes sense I guess. Any idea where this custom shape calculation would typically be specified? I've dug through the source code of Turi Create but to no avail.

**Comment by jiafatom** [comment]
hmm, there is an example in skl2onnx, see [here](https://github.com/onnx/sklearn-onnx/blob/0d3bc94ca95657dfa7f50f1b4fa421b498e017a2/tests/test_sklearn_voting_classifier_converter.py#L34-L48) for example.

**Comment by mbroedel** [comment]
Doing the same thing gives me an IndexError. Maybe the sound classifier is just written in such a way that it can't be converted easily.

**Comment by mpmontanez** [comment]
I've run into the same issue, but it's unclear to me how the customModel conversion function should be defined. Did you find a solution for this @mbroedel ?

**Comment by mbroedel** [comment]
Nope. But the turicreate sound classification toolkit isn't all that great anyways, so you can build one on your own that has comparable performance with relative ease using Keras and VGGish.

**Comment by mpmontanez** [comment]
Thanks @mbroedel - I'm trying to use turicreate for a recommender model. At this point, I'm looking into alternative tools as well.
